AirportTicketNookingTest/FilterFlightsShould.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Xunit;
+using Airport_Ticket_Booking;
+using System;
+using static Airport_Ticket_Booking.Common;
+
+namespace AirportTicketNookingTest
+{
+    public class FilterFlightsShould
+    {
+        private readonly FlightsManager _flightManager;
+
+        public FilterFlightsShould() {
+            _flightManager = FlightsManager.Instance();
+            _flightManager.ReadFromCSV(@"Flights.csv", true);
```

Comment:
We read this file at every test class which could cost performance if the file is large. Can you find a way in which we can read it only one time, something like making a shared context between tests?